src/multiplayer-servers/getting-started/using-the-agones-sdk.md

```diff
@@ -80,3 +80,22 @@ Shutdown() instead. This will cause the hosting environment to properly clean up
 game server and send a termination signal to the process, which can then be used to
 shut it down.
 
+## Player count and capacity tracking
+
+Game Servers created by GameFabric have a counter for players by default which you can use to track player numbers and capacity per game server.
```

Comment:
We use the writing "game server" (see for example [this docs page](https://docs.gamefabric.com/multiplayer-servers/getting-started/running-your-game-server#running-your-game-server)), so just the first letter of the sentence should be capitalized
```suggestion
Game servers created by GameFabric have a counter for players by default which you can use to track player numbers and capacity per game server.
```